subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/dsl/DefaultComponentMetadataProcessor.java

```diff
@@ -70,7 +70,8 @@
 
 public class DefaultComponentMetadataProcessor implements ComponentMetadataProcessor {
 
-    private final static boolean FORCE_REALIZE = Boolean.getBoolean("org.gradle.integtest.force.realize.metadata");
+    private final static boolean FORCE_REALIZE = true;
```

Comment:
:x: You didn't want to do this :)

Reply:
No I did not - damn forgot to revert that.